tscompare/methods.py

```diff
@@ -32,12 +32,17 @@
 import tskit
 
 
-def node_spans(ts):
+def node_spans(ts, include_missing=False):
```

Comment:
We weren't testing this option; I've added a test for that. However, we still don't have a test on a simple tree sequence where we know the right answer (instead of just comparing to the python version). Can you add a test of `node_spans` to those?